tetrateci/tsb-util/tsb-objects/group.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: gateway.tsb.tetrate.io/v2
+kind: Group
+metadata:
+  organization: {{ orgName }}
+  name: {{ gatewayGroupName }}
+  workspace: {{ workspaceName }}
+  tenant: {{ tenantName }}
+spec:
+  namespaceSelector:
+    names:
+      - "demo/bookinfo"
```

Comment:
change this to "{{ clusterName }}/{{ ns }}" -- For multi namespace app, this should include all the namespaces

Reply:
How do I obtain the clusterName variable